controls/KEYC-01-000005.rb

```diff
@@ -35,4 +52,25 @@
   tag stig_id: "KEYC-01-000005"
   tag cci: ["CCI-000018"]
   tag nist: ["AC-2 (4)"]
+
+  program = "/opt/keycloak/bin/kcadm.sh get events/config -r #{input('keycloak_realm')}"
+  
+  # Using inputs, testing inclusion of single element in array
+  describe json(content: command(program).stdout) do
+	  its('eventsEnabled') { should eq input('events_enabled') }
+	  its('eventsListeners') { should cmp input('events_listeners') }
+	  # Should this also include CLIENT_REGISTER?
+	  its('enabledEventTypes') { should include input(['enabled_event_types', 'value'], value: "REGISTER") }
+	  its('adminEventsEnabled') { should eq input('admin_events_enabled') }
+	  its('adminEventsDetailsEnabled') { should eq input('admin_events_details_enabled') }
+  end
+
+  # Using inputs, testing inclusion of entire array
+  # describe json(content: command(program).stdout) do
+	#   its('eventsEnabled') { should eq input('events_enabled') }
+	#   its('eventsListeners') { should cmp input('events_listeners') }
+	#   its('enabledEventTypes') { should cmp input('enabled_event_types') }
```

Comment:
I would lean towards only making the test check the one relevant event type for this control rather than the whole array, because what happens if this control fails? How do you know it is exactly this control that is the problem? Rather than a different event type.